x-pack/filebeat/module/cylance/protect/test/generated.log-expected.json

```diff
@@ -0,0 +1,3452 @@
+        "user.name": [
+            "onu"
+        ]
+    },
+    {
+        "@timestamp": "2017-06-26T09:42:33.000Z",
+        "event.category": "ZoneAddDevice",
+        "event.code": "CylancePROTECT",
+        "event.dataset": "cylance.protect",
+        "event.module": "cylance",
+        "event.original": "2017-6-26T7:42:33.mipsamvo eiusmod3517.internal.invalid CylancePROTECT oreveri ehende [eaqueip] Event Type: AuditLog, Event Name: ZoneAddDevice, Message: Device: olup; SHA256: labor, User: dol sciun (metcons) ",
+        "fileset.name": "protect",
+        "host.name": "eiusmod3517.internal.invalid",
+        "input.type": "log",
+        "log.offset": 9473,
+        "observer.product": "Protect",
+        "observer.type": "Anti-Virus",
+        "observer.vendor": "Cylance",
+        "rsa.identity.firstname": "dol",
+        "rsa.identity.lastname": "sciun",
+        "rsa.internal.messageid": "CylancePROTECT",
+        "rsa.investigations.event_cat": 1901000000,
+        "rsa.investigations.event_cat_name": "Other.Default",
+        "rsa.investigations.event_vcat": " AuditLog",
+        "rsa.misc.checksum": "labor",
+        "rsa.misc.event_type": "ZoneAddDevice",
+        "rsa.misc.mail_id": "metcons",
+        "rsa.misc.node": "olup",
+        "rsa.network.alias_host": [
+            "eiusmod3517.internal.invalid"
+        ],
+        "rsa.time.event_time": "2017-06-26T09:42:33.000Z",
+        "service.type": "cylance",
+        "tags": [
+            "cylance.protect",
+            "forwarded"
+        ]
+    },
+    {
+        "@timestamp": "2017-07-11T04:45:07.000Z",
+        "event.category": "DeviceRemove",
+        "event.code": "CylancePROTECT",
+        "event.dataset": "cylance.protect",
+        "event.module": "cylance",
+        "event.original": "11-July-2017 02:45:07 low oloreseo5039.test derit <<orese>dolor 2017-7-11T2:45:07.econs ntexpl3889.www.home CylancePROTECT yCic nder [mdolore] Event Type: Cic, Event Name: DeviceRemove, Device Name: saqu, Agent Version: iscive, IP Address: (10.156.34.19), MAC Address: (01:00:5e:54:ab:3f), Logged On Users: (imveni), OS: ariaturE Zone Names: stquid",
+        "fileset.name": "protect",
+        "host.name": "ntexpl3889.www.home",
+        "input.type": "log",
+        "log.offset": 9683,
+        "observer.product": "Protect",
+        "observer.type": "Anti-Virus",
+        "observer.vendor": "Cylance",
+        "related.ip": [
+            "10.156.34.19"
+        ],
+        "rsa.db.index": "stquid",
+        "rsa.internal.messageid": "CylancePROTECT",
+        "rsa.investigations.event_cat": 1804020000,
+        "rsa.investigations.event_cat_name": "Network.Devices.Removals",
+        "rsa.investigations.event_vcat": "Cic",
+        "rsa.misc.OS": "ariaturE",
+        "rsa.misc.event_type": "DeviceRemove",
+        "rsa.misc.node": "saqu",
+        "rsa.network.alias_host": [
+            "ntexpl3889.www.home"
+        ],
+        "rsa.network.eth_host": "01:00:5e:54:ab:3f",
```

Comment:
I'm assuming this is the network mac, could we add this under the `host.mac` field?